storage/ndb/rest-server/rest-api-server/internal/config/init_defaults.go

```diff
@@ -0,0 +1,88 @@
+/*
+ * This file is part of the RonDB REST API Server
+ * Copyright (c) 2022 Hopsworks AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package config
+
+import (
+	"os"
+	"sync"
+
+	"hopsworks.ai/rdrs/internal/log"
+)
+
+var globalConfig AllConfigs
+var mutex sync.Mutex
+
+func init() {
+	configFile := os.Getenv(CONFIG_FILE_PATH)
+	err := SetFromFileIfExists(configFile)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func newWithDefaults() AllConfigs {
+	return AllConfigs{
+		Internal: Internal{
+			BufferSize:          320 * 1024,
+			GOMAXPROCS:          -1,
+			PreAllocatedBuffers: 1024,
+		},
+		GRPC: GRPC{
+			ServerIP:   "localhost",
+			ServerPort: 4406,
+		},
+		REST: REST{
+			ServerIP:   "localhost",
+			ServerPort: 5406,
+		},
+		RonDB: RonDB{
+			Mgmds: []Mgmd{
+				{
+					IP:   "localhost",
+					Port: 1186,
+				},
+			},
+		},
+		MySQL: MySQL{
+			Servers: []MySQLServer{
+				{
+					IP:   "localhost",
+					Port: 3306,
+				},
+			},
+			User:     "rondb",
+			Password: "rondb",
+		},
+		Security: Security{
+			EnableTLS:                        false,
```

Comment:
true

Reply:
I see that there is a point of setting this true for the sake of testing it in unit tests.

However, this also means that we have a default setting that will fail when not running unit tests. Since `CertificateFile` & `PrivateKeyFile` are empty by default and are only generated for unit tests.

Reply:
Changed it anyways